poc.js

```diff
@@ -1,212 +1,211 @@
 const MongoClient = require('mongodb').MongoClient;
 const elasticsearch = require('elasticsearch');
-const esClient = new elasticsearch.Client({ host: 'localhost:9200', apiVersion: '2.4' });
+const esClient = new elasticsearch.Client({host: 'localhost:9200', apiVersion: '2.4'});
 const fs = require('fs');
 const f = require('util').format;
-const BSON = require('bson')
+const BSON = require('bson');
 const bson = new BSON();
 let url = process.env.MONGO_RS || 'mongodb://localhost:27017';
 const dbName = 'vrs';
-const collectionName = 'voter';
 let resumeToken;
-const ARG_SKIP = Number(process.argv[2]) || 0;
-const ARG_LIMIT = Number(process.argv[3]) || 100000;
-const ARG_DUMP = process.argv[4] === '-d' ? true : false;
+
+//1st, 2nd, & 3rd command line args:
+const ARG_SKIP = Number(process.argv[2]) || 0; // Set to skip N documents ahead in your query
+let ARG_LIMIT = Number(process.argv[3]) || 100000; // the bulk insertion limit
```

Comment:
Why wouldn't this also be `const`? If it really does need to change later, I'd suggest capturing the original arg as a constant, and then initializing a separate variable to it later.